EstateManagement/Startup.cs

```diff
@@ -69,20 +80,68 @@
             Startup.Configuration = builder.Build();
             Startup.WebHostEnvironment = webHostEnvironment;
         }
-        
+
+        /// <summary>
+        /// Gets or sets the configuration.
+        /// </summary>
+        /// <value>
+        /// The configuration.
+        /// </value>
         public static IConfigurationRoot Configuration { get; set; }
 
+        /// <summary>
+        /// Gets or sets the web host environment.
+        /// </summary>
+        /// <value>
+        /// The web host environment.
+        /// </value>
         public static IWebHostEnvironment WebHostEnvironment { get; set; }
 
+        private static EventStoreClientSettings EventStoreClientSettings;
+
+        private void ConfigureEventStoreSettings(EventStoreClientSettings settings = null)
+        {
+            if (settings == null)
+            {
+                settings = new EventStoreClientSettings();
+            }
+
+            settings.CreateHttpMessageHandler = () => new SocketsHttpHandler
+                                                      {
+                                                          SslOptions =
+                                                          {
+                                                              RemoteCertificateValidationCallback = (sender,
+                                                                                                     certificate,
+                                                                                                     chain,
+                                                                                                     errors) => true,
+                                                          }
+                                                      };
+            settings.ConnectionName = Startup.Configuration.GetValue<String>("EventStoreSettings:ConnectionName");
+            settings.ConnectivitySettings = new EventStoreClientConnectivitySettings
+                                            {
+                                                Address = new Uri(Startup.Configuration.GetValue<String>("EventStoreSettings:ConnectionString")),
+                                            };
+
+            settings.DefaultCredentials = new UserCredentials(Startup.Configuration.GetValue<String>("EventStoreSettings:UserName"),
+                                                              Startup.Configuration.GetValue<String>("EventStoreSettings:Password"));
+            Startup.EventStoreClientSettings = settings;
```

Comment:
Codacy found an issue: [Make the enclosing instance method 'static' or remove this set on the 'static' field.](https://app.codacy.com/manual/stuart_ferguson1/EstateManagement/pullRequest?prid=6232863)